tests/std/tests/P2387R3_bind_back/test.cpp

```diff
@@ -101,6 +99,33 @@ constexpr bool test_constexpr() {
     assert(move(bound5)() == "modifiable rvalue");
     assert(move(as_const(bound5))() == "const rvalue");
 
+    // Test PMFs.
+    Cat cat{"Peppermint"};
+    auto bound2 = bind_back(&Cat::noise, "meow");

```

Comment:
The order here is strange - the `bound` variables are named 0, 1, 5, 2, 3, 6, 7, 8, 9. It would make more sense to move the `bound5` "perfect forwarding call wrapper" case after the PMFs, which is the order used by the `bind_front` test.